retroachievements/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+
+RetroAchievements API Wrapper
+---
+
+Official Python API wrapper for RetroAchievements
```

Comment:
"wrapper" or "client"?

Reply:
Good catch. The docs will always refer to this as a client library.

Reply:
At this point I'd call it a wrapper, it's just spitting out the HTTP response in a format that Python applications can make use of. If you want I can change it to "client"

Reply:
Personally, I prefer the term "client" - not a strong opinion though. In this case it would be fitting for consistency's sake.

Reply:
Let's go with client -- it's plausible this package could also do some sane type-setting kind of like api-js does, for example:

```
// HTTP API
"Untracked": 0
```

```
// api-js
"untracked": false
```

Reply:
Description updated